docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -160,7 +186,12 @@ To use the Bluetooth functionality of the device, you must switch the Bluetooth
    adapter.setName(chatServerName);
    ```
 
-## Discovering Bluetooth Devices
+## Managing remote devices
+
+<!-- TODO #02
+      - write about method isConnected()
```

Comment:
isConnected() is a method of BluetoothLEDevice and is only relevant for BLE devices. I see, it's in the section about BLE, so please, just remove it from here.